optaplanner-core/src/main/java/org/optaplanner/core/config/score/director/ScoreDirectorFactoryConfig.java

```diff
@@ -156,6 +174,35 @@ public void setScoreDrlFileList(List<File> scoreDrlFileList) {
         this.scoreDrlFileList = scoreDrlFileList;
     }
 
+    public Boolean getCompileDroolsAlphaNetwork() {
+        return compileDroolsAlphaNetwork;
+    }
+
+    public void setCompileDroolsAlphaNetwork(Boolean compileDroolsAlphaNetwork) {
+        this.compileDroolsAlphaNetwork = compileDroolsAlphaNetwork;
+    }
+
+    public boolean isDroolsAlphaNetworkCompilerEnabled() {
```

Comment:
This method is used from Quarkus/Spring boot code injection.
I don't particularly like that it is here, but it is the only reasonable way I could think of to share this code.

Reply:
In any case, let's not put it between the getters and setters, but before or after all the "simple getters and setters". These days we put the "simple getters and setters" at the end because it's just Java ceremony.